### PR TITLE
[snet-stakers] strategy: snet-stakers

### PR DESCRIPTION
**Pull request by Vivek205**
This strategy returns the staking pool balances of the voters for the SingularityNET project.

### CODE REVIEW

File: src/strategies/snet-stakers/index.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'bonustrack';
```
Comment (ChaituVR) [inline src/strategies/snet-stakers/index.ts:5]:
```suggestion
export const author = 'Vivek205 ';
```

File: src/strategies/snet-stakers/index.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+export const version = '0.1.1';
```
Comment (ChaituVR) [inline src/strategies/snet-stakers/index.ts:6]:
```suggestion
export const version = '0.1.0';
```

File: src/strategies/snet-stakers/index.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = 'latest';
```
Comment (ChaituVR) [inline src/strategies/snet-stakers/index.ts:45]: Same here https://github.com/snapshot-labs/snapshot-strategies/pull/202#discussion_r758960400

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/snet-stakers/index.ts (hunk 3 above)
Comment (Vivek205) [reply src/strategies/snet-stakers/index.ts]: fixed in https://github.com/snapshot-labs/snapshot-strategies/pull/203/commits/732e6e477b32e5918ba826bd1604f3ae6c244237

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**